pkgs/tools/misc/tmuxp/default.nix

```diff
@@ -6,11 +6,11 @@ let
 in
 pypkgs.buildPythonApplication rec {
   pname = "tmuxp";
-  version = "1.9.2";
+  version = "1.11.0";
 
   src = pypkgs.fetchPypi {
     inherit pname version;
-    sha256 = "sha256-3RlTbIq7UGvEESMvncq97bhjJw8O4m+0aFVZgBQOwkM=";
+    sha256 = "sha256-N5kZ+e17ZgLOCvV/lcT/hdG1VNqLxh98QOQyM0BmZCA=";
   };
 
   # No tests in archive
```

Comment:
But there are tests on GitHub. Can we run those?

Reply:
I am not intending to change how the package is done but upgrade it to fix the build error now in master.

Reply:
I think that's fine for now. Still strictly better